Cargo.toml

```diff
@@ -65,9 +65,9 @@ atty = "0.2"
 dirs = "4.0"
 
 # binary dependencies
-enarx-shim-kvm = { path = "src/bin/shim-kvm", artifact = "bin", target = "x86_64-unknown-none" }
-enarx-shim-sgx = { path = "src/bin/shim-sgx", artifact = "bin", target = "x86_64-unknown-none" }
-enarx-exec-wasmtime = { path = "src/bin/exec-wasmtime", artifact = "bin", target = "x86_64-unknown-linux-musl" }
+enarx-shim-kvm = { version = "0.4.0-pre", artifact = "bin", target = "x86_64-unknown-none" }
```

Comment:
I thought a version is not necessary and Cargo will do it automatically

Reply:
I thought cargo makes a copy of the Cargo.toml and fills out the actual versions for the crates, which are only specified by the path. Try with cargo package and look into the generated Cargo toml

Reply:
```
> cargo publish --dry-run
error: all dependencies must have a version specified when publishing.
dependency `enarx-exec-wasmtime` does not specify a version
Note: The published dependency will use the version from crates.io,
the `path` specification will be removed from the dependency declaration.
```
In order to publish, we do need a version field. What we don't necessarily need to do is remove the path field. However, I have chosen to remove the path field here in order to force CI to run a full test using only the versions on crates.io, rather than allowing it to use the local copies of the crates.